src/lib/Breadcrumbs.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import Link from './Link';
+
+import * as palette from '../palette';
+
+const StyledNav = styled.nav`
+  font-family: ${palette.FONT.main};
+
+  height: ${props => props.shadow ? '45px' : 'auto'};
+  border-radius: ${props => props.shadow ? '5px' : 'none'};
+  box-shadow: ${props => props.shadow ? palette.SHADOW : 'none'};
+  
+  line-height: ${props => props.shadow ? '45px' : 'normal'};
```

Comment:
I'd stick with the `${p => p.shadow [...]` convention, it's more succinct.

Reply:
OK